source/dev_guide/dev/dev_guide_technical_standards_for_core_contrib.rst

```diff
@@ -75,6 +75,86 @@ Standardised Header
    - It is not necessary to specify exact :ref:`code/array structure <dev_code_contribution_standardVariables>` (just :code:`<CODE>` or :code:`<ARRAY>`). However, a specific structure makes it clear what is expected/returned.
    - It is not normal to have 10 arguments. The majority of humans can only `subitise <https://www.dictionary.com/browse/subitize>`_ ~5 items, use that as a guide.
 
+.. _dev_code_contribution_standard_config:
+
+Standardised Config Classes
+============================================================
+
+.. card::
+   :class-card: sd-card-2
+   :class-header: header-2-light
+
+   Standardised Config Classes
+   ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+   .. rst-class:: code-block-2
+   .. code-block::
+      //Example - config.hpp //{modset - filename}
```

Comment:
please add empty line before this line, otherwise it won't compile to the page:

```
/$basepath/A3-Antistasi-Docs/source/dev_guide/dev/dev_guide_technical_standards_for_core_contrib.rst:91: ERROR: Error in "code-block" directive:
maximum 1 argument(s) allowed, 6 supplied.

.. code-block::
   //Example - config.hpp //{modset - filename}

   /*
   Maintainer: John Doe
       Showing example of how to patch the baseWeapon attribute of a class.
       Showing example of how a3a classes should be written.
   */

   class CfgPatches
   {
      //Use a relevant patchname, A3 is used for vanilla config fixes.
      //For example "AirLoadout_Vanilla" is used for the vanilla aircraft loadouts.
      class PATCHNAME(A3)
      {
         name = COMPONENT_NAME;
         units[] = {};
         weapons[] = {};
         requiredVersion = REQUIRED_VERSION;
         requiredAddons[] = {"A3_Data_F_AoW_Loadorder"};    //In most cases any patch from the required mod(s) is sufficent - though be careful when overwriting classes.
         skipWhenMissingDependencies = 1;    //For most A3A uses we want to skip configs that don't have all their requiredAddons.
         author = AUTHOR;
         authors[] = { AUTHORS };
         authorUrl = "";
         VERSION_CONFIG;
      };
   };

   class CfgWeapons
   {
      class arifle_AK12_GL_base_F;
      class arifle_AK12_GL_lush_F : arifle_AK12_GL_base_F
      {
         baseWeapon = "arifle_AK12_GL_lush_F";
      };
   };

   class CfgVehicles
   {
      //Grandparent
      class rhs_t72ba_tv;

      //Parent and child class
      class rhsgref_ins_t72ba : rhs_t72ba_tv {class EventHandlers; };

      // a3a_{modset}_{identifier}
      class a3a_rhs_chdkz_72a : rhsgref_ins_t72ba
      {
         //Inherits the eventhandler from the rhsgref_ins_t72ba.
         class EventHandlers : EventHandlers
         {
            //Clears the eventhandler that adds a flag to the tank
            class rhs_flag_init{};
         };
         //it is also possible to retexture vehicles in here using hidden selections. See githuh for `examples <https://github.com/official-antistasi-community/A3-Antistasi/tree/unstable/A3A/addons/config_fixes>`_.
         //hiddenSelectionsTextures[] = {};
      };
   };
```